api/clients/v2/accountant.go

```diff
@@ -157,9 +157,24 @@ func (a *Accountant) SymbolsCharged(numSymbols uint64) uint64 {
 	return core.RoundUpDivide(numSymbols, a.minNumSymbols) * a.minNumSymbols
 }
 
+// GetRelativePeriodRecord returns the period record for the given index
+// return empty record if there is no record for the relative index
```

Comment:
Is this comment about the edge case where `relativeIndex >= uint32(a.numBins)`?

Reply:
This is no longer true with the panic; did you remove the comment?